server/server.go

```diff
@@ -32,6 +36,43 @@ func (r *rodeServer) BatchCreateOccurrences(ctx context.Context, occurrenceReque
 	}, nil
 }
 
+func (r *rodeServer) AttestPolicy(ctx context.Context, request *pb.AttestPolicyRequest) (*pb.AttestPolicyResponse, error) {
+	log := r.logger.Named("AttestPolicy")
+	log.Debug("received requests")
+
+	// check OPA policy has been loaded
+
+	// fetch occurrences from grafeas
+	listOccurrencesResponse, err := r.grafeasClient.ListOccurrences(ctx, &grafeas.ListOccurrencesRequest{Filter: fmt.Sprintf("resourceUri:%s", request.ResourceURI)})
+	if err != nil {
+		log.Error("failed to list occurrences for resource", zap.NamedError("error", err), zap.String("resource", request.ResourceURI))
+		return nil, err
+	}
+
+	// json encode occurrences
+	_, err = json.Marshal(listOccurrencesResponse)
```

Comment:
you'll want to use the `jsonpb` package to do marshalling / unmarshalling of protobufs